sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/events/AddParticipantsFailedEvent.java

```diff
@@ -48,4 +55,13 @@ private AddParticipantsFailedEvent(@JsonProperty("participants") List<Map<String
     public List<CommunicationIdentifier> getParticipants() {
         return this.participants;
     }
+
+    /**
+     * Get the resultInformation property: Contains the resulting SIP code/sub-code and message from NGC services.
+     *
+     * @return the resultInformation value.
+     */
+    public ResultInformation getResultInformation() {
```

Comment:
Quick question - shouldn't ResultInformation be in base class? since it is common among other events

Reply:
Hmm - you are actually right. .NET side includes it as common among all events, I am curious what was the reason behind that. We can revisit this after.